src/crypto.d.ts

```diff
@@ -0,0 +1,183 @@
+interface AesCbcParams extends Algorithm {
+  name: AesAlgorithm
+  iv: BufferSource
```

Comment:
Can you clarify a bit more about what your starting source was and what your process was?

The corresponding interface in https://github.com/microsoft/TypeScript/blob/master/lib/lib.webworker.d.ts for instance seems to be spelling out the union type for various arrays rather than using ArrayBufferView